ABloom/ABloom/Firebase/Model/DBUserModel.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  DBUserModel.swift
+//  ABloom
+//
+//  Created by 정승균 on 10/29/23.
+//
+
+import Foundation
+/// DB에 저장될 User Model
+struct DBUser: Codable {
+  let userId: String
+  let name: String?
+  let sex: Bool?
+  let estimatedMarriageDate: Date?
```

Comment:
```suggestion
  let marriageDate: Date?
```
이 부분 변수를 수정하는 건 어떨까요?? 공수가 많이 들까요??

Reply:
좋아요!